quality_control_stock/models/qc_inspection.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2014 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class QcInspection(models.Model):
+    _inherit = 'qc.inspection'
+
+    @api.multi
+    def object_selection_values(self):
+        result = super(QcInspection, self).object_selection_values()
+        result.extend([('stock.picking', "Picking List"), ('stock.move', "Stock Move")])
+        return result
+
+    @api.multi
+    @api.depends('object_id')
+    def _compute_picking(self):
+        for inspection in self:
+            inspection.picking_id = False
```

Comment:
Computed values are False by default before compute call. So, if value is not set, False will be.